### DISCUSSION TITLE
french regional tv ...

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#141] [Show and tell]
tele nantes
https://vialmtv.tv/tele-nantes/
c8
https://vialmtv.tv/c8-en-direct/
tele grenoble 
https://vialmtv.tv/tele-grenoble/
television loire 7
https://vialmtv.tv/television-loire-7/
tv vendée
https://vialmtv.tv/tv-vendee/
lm tv sarthe
http://www.lmtv.fr/

**Comment by MapGuy11** [comment]
Please make an issue so we can address it easier. 

Thanks

**Reply by ARAB-IPTV** [reply]
okay i will open an issue for every channel
